fixes DOCS-159 (#248)
pinot-contrib/pinot-docs: configuration-reference/table.md

| invertedIndexColumns                       | The list of columns that inverted index should be created on. The name of columns should match the schema. e.g. in the table above, inverted index has been created on three columns `foo`, `bar`, `moo`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |

> | createInvertedIndexDuringSegmentGeneration | Boolean to indicate whether to create inverted indexes during the segment creation. By default, false i.e. inverted indexes are created when the segments are loaded on the server. Learn more about this setting in [advanced Pinot setup](../developers/advanced/advanced-pinot-setup.md#add-an-inverted-index-to-your-table-automatically).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |

Author: I only added the sentence with the link. If I started editing the text of the cell, I would find myself editing the text across the entire page. That's a big job for another day.

> | createInvertedIndexDuringSegmentGeneration | Boolean to indicate whether to create inverted indexes during the segment creation. By default, false i.e. inverted indexes are created when the segments are loaded on the server. Learn more about this setting in [advanced Pinot setup](../developers/advanced/advanced-pinot-setup.md#add-an-inverted-index-to-your-table-automatically).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |

Reviewer: ```suggestion
| createInvertedIndexDuringSegmentGeneration | Boolean to indicate whether to create inverted indexes when segments are created. By default, false, which means indexes are created when the segments are loaded on the server. Learn more about this setting in [advanced Pinot setup](../developers/advanced/advanced-pinot-setup.md#add-an-inverted-index-to-your-table-automatically).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Reviewer: to be sure, false means indexes (including this inverted one) are created when segments are loaded on the server.....which is different than true, which means the inverted index is created when a segment is created.

If the above is true, think we can rewrite a bit to clarify, as false sounds like it might be true
